public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx

```diff
@@ -325,12 +326,12 @@ export const VisaulizationFlyout = ({
           <EuiFlexItem>
             {previewLoading ? (
               <EuiLoadingChart size="xl" mono className="visualization-loading-chart-preview" />
-            ) : isPreviewError !== '' ? (
-              <div className="visualization-error-div-preview">
+            ) : !_.isEmpty(isPreviewError) ? (
+              <div className="visualization-error-div">
                 <EuiIcon type="alert" color="danger" size="s" />
                 <EuiSpacer size="s" />
                 <EuiText size="s">
-                  <p>{isPreviewError}</p>
+                  <p>{isPreviewError.error}</p>
```

Comment:
is this supposed to be errorMessage or errorDetails?

Reply:
Thanks for the catch missed this. Updated : https://github.com/opensearch-project/dashboards-observability/pull/247/commits/c46c072d78eae320e585105418d70cedd86e7514